Automattic/woocommerce-payments
# Default WooPay first party feature flag to on

Fixes #Automattic/woopay#2189 

#### Changes proposed in this Pull Request

This PR just defaults the `_wcpay_feature_woopay_first_party_auth` feature flag to be on. This is part of the larger WooPay First Party Authentication project and is closely related to #7199. 

#### Testing instructions

The testing instructions would be the same as in https://github.com/Automattic/woopay/pull/2160. The only difference is to make sure that `_wcpay_feature_woopay_first_party_auth` option is deleted from the `wp_options` table before testing to confirm that it defaults to First Party Authentication flow.

In addition to testing the first party authentication flow steps, since this is changing a default feature flag we should also do regression testing covering at least:
- Multiple payment request methods (Google Pay and Apple Pay)
- Testing checkout on shortcode and blocks
- Testing with UPE and non-UPE enabled
- Testing other payment methods as part of UPE (so non-credit cards)

-------------------


**Post merge**
